packages/react-fiche-service-public/src/components/ElementBuilder.js

```diff
@@ -22,11 +22,15 @@ const parseChildren = (children, headingLevel) => (
 
 // Beware, this one is recursive
 export function ElementBuilder({ data, headingLevel }) {
+  // Sometime we get elements without children...
+  if (!data) return null;
```

Comment:
+1 to add a comment to the why but also adding the ref to current data could be useful too